### ISSUE TITLE
FMM and dense are giving different results for Helmholtz with complex wavenumber

### ISSUE THREAD

**Issue by mscroggs** [bempp/bempp-cl#20]
The assertion in this minimal example fails:

```
import numpy as np
import bempp.api
from bempp.api import function_space
from bempp.api.operators.boundary import helmholtz

wavenumber = 2.5 + 1j
grid = bempp.api.shapes.regular_sphere(2)
space = function_space(grid, "DP", 0)

op1 = helmholtz.single_layer(space, space, space, wavenumber, assembler="dense")
op2 = helmholtz.single_layer(space, space, space, wavenumber, assembler="fmm")

fun = bempp.api.GridFunction(space, coefficients=np.random.rand(space.global_dof_count))

assert np.allclose((op1 * fun).coefficients, (op2 * fun).coefficients)
```

**Comment by tbetcke** [comment]
Support for complex wavenumbers was just added in Exafmm and is not yet extended to the interface. Will need to do this.

**Comment by mscroggs** [comment]
This is fixed in #45
